lib/scripts/eventPage.js

```diff
@@ -8,6 +8,43 @@ var contextMenu = new ContextMenu(),
  * Set up default data and store it in `chrome.storage.localData`
  */
 function initializeLocalStorage() {
+  let localData = {
```

Comment:
@rohankatyal29 `localData` is already imported in this line `import { localData } from './utils/defaultStorage';`. All that has to be done here is to set it. Can you remove this? If you want to add anything to local storage, check the `defaultStorage.js` in `services`